lib/util.js

```diff
@@ -16,6 +16,8 @@ const { logger } = require('./logging');
 const somaDOM = require('./soma_dom');
 const ImageAsset = require('./asset/imageAsset');
 const VideoAsset = require('./asset/videoAsset');
+const NewsArticle = require('./asset/newsArticle');
+const BlogArticle = require('./asset/blogArticle');
```

Comment:
Please keep these in alphabetical order

Reply:
Will do. This is not being checked in our lint rules. Do you think we should enable https://eslint.org/docs/rules/sort-imports ?

Reply:
Yeah, I would like to. However I'd move linting cleanups to a different PR.